iocore/hostdb/HostDB.cc

```diff
@@ -1588,6 +1588,12 @@ HostDBContinuation::set_check_pending_dns()
   this->setThreadAffinity(this_ethread());
   HostDBContinuation *c = q.head;
   for (; c; c = static_cast<HostDBContinuation *>(c->link.next)) {
```

Comment:
You want to see `this` continuation is in the `Queue<HostDBContinuation> &q`?
You can avoid this for-loop by using `q.in(this)`, I think.

Reply:
@masaori335 Good idea!

Reply:
fixed!